src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
+
+Array.from(document.querySelectorAll('.stars')).forEach(function (a) {
+  a.addEventListener('click', function (e) {
+    e.preventDefault();
+    console.log(e);
+    e.originalTarget.classList.add('full');
+    var previousSibling = e.originalTarget.nextElementSibling;
+    var nextSibling = e.originalTarget.previousElementSibling;
+    // Direction jest ustawione na rtl wiec next jest tak naprawde previous ;) (i odwrotnie)
+    while (previousSibling) {
```

Comment:
Please remove `voted` class only when needed. 
You can select only `voted` class elements.